business/src/main/java/com/picsart/business/feature/posts/data/datasource/NetworkGetPostsDataSource.kt

```diff
@@ -13,5 +23,13 @@ internal class NetworkGetPostsDataSource(
     private val endpoint: String,
     private val httpClient: HttpClient
 ) : GetListDataSource<PostDTO> {
-    override suspend fun invoke(): Either<Failure, List<PostDTO>> = TODO()
+    override suspend fun invoke(): Either<Failure, List<PostDTO>> =
```

Comment:
In this moment maybe it would be nice to explain the advantages + explanation of Either and show them the Implementation

Reply:
Good point, will add explanation 👍🏻